### PR TITLE
RavenDB-19933 Revamp Add Shard modal; minor styling fixes

### PR DESCRIPTION
**Pull request by mateuszbartosik**
### Issue link
https://issues.hibernatingrhinos.com/issue/RavenDB-19933/Revamp-Add-Shard-modal

### Additional description
Revamp `Add Shard` modal; minor styling fixes for buttons, modals, stats view

### Type of change
- Optimization

### How risky is the change?
- Not relevant

### Backward compatibility
- Not relevant

### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing by Contributor
- It has been verified by manual testing

### Testing by RavenDB QA team
- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- UI work needed

### CODE REVIEW

File: src/Raven.Studio/typescript/components/pages/database/status/statistics/partials/StatsHeader.tsx [hunk 1]
```diff
@@ -19,21 +19,22 @@ export function StatsHeader() {
...
                 <Col sm="auto">
                     <Button
-                        color="primary"
+                        color="secondary"
                         onClick={() => dispatch(toggleDetails())}
                         title="Click to load detailed statistics"
                     >
                         <i className={classNames(detailsVisible ? "icon-collapse-vertical" : "icon-expand-vertical")} />
-                        <span>{detailsVisible ? "Hide" : "Show"} details</span>
+                        <span>{detailsVisible ? "Hide" : "Show"} detailed database & indexing stats</span>
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/status/statistics/partials/StatsHeader.tsx]: `&` -> `&amp;`

File: src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html [hunk 2]
```diff
@@ -1,26 +1,18 @@
 <div class="modal-dialog add-new-node-to-db-group" role="document">
-    <div class="modal-content" tabindex="-1">
+    <div class="modal-content neo-modal" tabindex="-1">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html:2]: did we already defined neo-modal somewhere?

File: src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html [hunk 3]
```diff
@@ -1,26 +1,18 @@
...
-            <div class="flex-form">
-                <div class="form-group" data-bind="validationElement: replicationFactor">
-                    <div>
-                        <label class="control-label">Replication factor</label>
-                    </div>
-                    <div class="flex-grow">
-                        <input class="form-control" type="number" min="1" placeholder="Enter number of nodes" data-bind="numericInput: replicationFactor">
-                    </div>
+            <h4>Replication Configuration <i class="icon-info" title="Set the number of nodes that will contain the database"></i></h4>
+            <!-- TODO: Add context -->
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html:11]: You wrote in PR summary that no UI work in needed. ? ;-)

File: src/Raven.Studio/wwwroot/Content/css/styles-common-scoped.less [hunk 4]
```diff
@@ -2157,6 +2157,14 @@ ul.properties {
...
 }
 
+.align-items-center {
+    align-items: center;
+}
+
+.align-self-center {
+    align-self: center;
+}
+
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/styles-common-scoped.less:2167]: bs5 already provides that https://getbootstrap.com/docs/5.0/utilities/flex/#align-self ?

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/styles-common-scoped.less:2167]: ok this for bs3, so please ignore my comment

File: src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html (hunk 2 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html:2]: It is used in the `shardSelector` view

File: src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html (hunk 3 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/App/views/resources/addNewShardToDatabaseGroup.html:11]: Got me 🙈 This modal needs to be discussed later.

File: src/Raven.Studio/typescript/components/pages/database/status/statistics/partials/StatsHeader.tsx (hunk 1 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/typescript/components/pages/database/status/statistics/partials/StatsHeader.tsx]: Fixed

**Review by ml054 — APPROVED** [review]

**Comment by karmeli87** [comment]
Studio tests failed

**Comment by mateuszbartosik** [comment]
> Studio tests failed

@karmeli87 tests have been fixed

**Review by ml054 — APPROVED** [review]

**Pull request merged**